### ISSUE TITLE
Dataset Split to Train,Validation and Test

### ISSUE THREAD

**Issue by EnisBerk** [Yunhua468/Audio-Visual-Emotion-and-Sentiment-Research#3]
Hi Everyone, 
We need to divide our dataset into train, validation, and test, in the same way, to report the compatible results. For example, a sample in my training set should not be someones' test set otherwise we will get results that do not reflect true performance our model. Please, use the following CSV files for splitting the dataset. Let me know if you disagree. 

[DatasetSplitCSV.zip](https://github.com/Yunhua468/Audio-Visual-Emotion-and-Sentiment-Research/files/4594396/DatasetSplitCSV.zip)
public link for downloading easier: [dropbox](https://www.dropbox.com/s/qdhtexle4p0ngc3/DatasetSplitCSV.zip?dl=1)

@patrick-jean-baptiste @jjdld @Yunhua468

**Comment by Yunhua468** [comment]
Hi guys,

I think let us process the audio only files with the model including all the audio files, need to think if we need to separate the song and speech.
And let us process the video only files with the model including all the video files, also need to think if we need to separate the song and speech?
And for the files including video and audio part, let us use the split function on the file names, I think this is a better way. What do you guys think?

**Comment by EnisBerk** [comment]
Adding this comment here to move our offline discussion online. 

- There was a misunderstanding about file numbers that's why Yunhua talking about audio and video files together. Dataset provides each sample in 3 formats: audio, video, and video without sound. We do not need to do any extraction or merging.
- We decided to keep song and speech samples together rather than processing separately.

**Comment by EnisBerk** [comment]
@patrick-jean-baptiste @jjdld @Yunhua468 
how to download csv and load the audio files, you can modify file name to load images

```bash
wget -q dropbox.com/s/qdhtexle4p0ngc3/DatasetSplitCSV.zip?dl=1 -O DatasetSplitCSV.zip
unzip -q DatasetSplitCSV.zip
```

```python3

import pandas as pd
train=pd.read_csv("train.csv",header=None)
valid=pd.read_csv("valid.csv",header=None)
test=pd.read_csv("test.csv",header=None)

x_train=[]
x_val=[]
x_test=[]
for x in train[0]:
    embed=embeddings[params["embeddingType"]][x]
    x_train.append(embed)

for x in valid[0]:
    embed=embeddings[params["embeddingType"]][x]
    x_val.append(embed)

for x in test[0]:
    embed=embeddings[params["embeddingType"]][x]
    x_test.append(embed)

x_train=np.array(x_train)
x_val=np.array(x_val)
x_test=np.array(x_test)

y_train=np.array(train[1].astype('category').cat.codes)
y_val=np.array(valid[1].astype('category').cat.codes)
y_test=np.array(test[1].astype('category').cat.codes)
```

**Comment by Yunhua468** [comment]
Hey guys,

Pay attention to merge the dataset, “Modality (01 = full-AV, 02 = video-only, 03 = audio-only).” , so the file name is different.

Best Regards,
Yunhua

**Comment by Yunhua468** [comment]
split dataset filename can not match with images, images totally 2878

**Comment by patrick-jean-baptiste** [comment]
Hi,

The reason is that the video data comes with video only and audio-visual videos.
1440 for each of the two. Two images could did not have a face detected and were excluded.
This is just for the files with speech.

I downloaded the files for song as well so there should be approximately two images per one audio (since I used video only and audio-visual videos).

I am putting together the images. Will send soon.

Regards,
Patrick Jean-Baptiste
